### PR TITLE
Update uuid to remove deprecation warning + up version

### PR DESCRIPTION
**Pull request by mylesrankin**
There's a deprecation warning for the version of uuid that this package is using, therefore we should update uuid to the latest version.

I have also bumped the npm version on the package (to 1.0.2, as it shows as 1.0.1 on npm already).

Deprecation warning for reference:
```
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
```

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (tellnes) [inline package-lock.json]: Please do not commit the `package-lock.json` file as part of this PR

**Comment by tellnes** [comment]
Looks like I did forget to push the version bump commit I made after the previous release. Pushed it now. It did create a conflict with your version bump in `package.json`.
Please fix the conflict (you can keep the version bump in `package.json`) and remove the `package-lock.json` file and I will merge and publish this.

**Comment by mylesrankin** [comment]
@tellnes Done, is that OK?

**Comment by tellnes** [comment]
Released as `1.0.2`

**Pull request merged**